hack/csv-generate.sh

```diff
@@ -62,33 +75,37 @@ if [[ "$CSV_VERSION" != "$PREV" ]] && [[ "$CSV_VERSION" != "$OLD" ]]; then
   $OPERATOR_SDK generate csv \
     --operator-name="${PACKAGE_NAME}" \
     --csv-version="${CSV_VERSION}" \
-    --csv-channel="${CSV_VERSION}" \
+    --csv-channel="${CSV_CHANNEL}" \
     --default-channel=true \
     --update-crds \
     --make-manifests=false \
+    --from-version="${CSV_FROM_VERSION}" \
     --output-dir="${OUT_ROOT}"
 
+  # copy template CSV file to preserve it for our csv-generator
+  cp "${OUT_CSV_FILE}" "${CSV_TEMPLATE_FILE}"
```

Comment:
this is going to be overwritten by `csv-generator`, isn't it? if so, let's just `mv` it

Reply:
yep, done